server/controllers/admin/employees.js

```diff
@@ -44,30 +53,31 @@ exports.list = function (req, res, next) {
      JOIN debtor ON employee.debtor_uuid = debtor.uuid
      JOIN creditor ON employee.creditor_uuid = creditor.uuid
      JOIN creditor_group ON creditor_group.uuid = creditor.group_uuid
-     ORDER BY employee.name ASC, employee.postnom ASC, employee.prenom ASC;`;
+     ORDER BY employee.name ASC, employee.postnom ASC, employee.prenom ASC;
+  `;
 
   db.exec(sql)
   .then(function (rows) {
     res.status(200).json(rows);
   })
   .catch(next)
   .done();
-};
+}
 
 /**
  * Get list of availaible holidays for an employee
  */
 exports.listHolidays = function (req, res, next) {
-  var pp = JSON.parse(req.params.pp);
-  var sql =
+  let pp = JSON.parse(req.params.pp);
```

Comment:
Do you know what `pp` means? Not a big issue but this should be discouraged in the future or very well commented.